tests/framework/extensions/rke1/componentchecks/etcdversion.go

```diff
@@ -5,24 +5,46 @@ import (
 	"strings"
 
 	"github.com/rancher/rancher/tests/framework/clients/rancher"
-	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	nodes "github.com/rancher/rancher/tests/framework/pkg/nodes"
```

Comment:
super nitpick but we don't need the nodes package var.

Reply:
Resolved